oic.r.acl2.raml

```diff
@@ -110,15 +127,26 @@ traits:
         description: The request is invalid.
```

Comment:
yeah - it's now conntype of "auth-crypt" or "anon-clear". This example reflects CR-48. And this aspect was superceded. 

The examples don't get as much attention as they should.